VocaDbWeb/Controllers/StatsController.cs

```diff
@@ -228,22 +230,28 @@ private ActionResult SimplePieChart(string title, string seriesName, ICollection
 			});
 		}
 
-		private StatsQueries.LocalizedValue[] GetTopValues<T>(Func<IQueryable<T>, IQueryable<StatsQueries.LocalizedValue>> func)
+#nullable enable
+		private StatsQueries.LocalizedValue[] GetTopValues<T>(Func<IQueryable<T>, IQueryable<StatsQueries.LocalizedValue>> func, DateTime? cutoff)
 			where T : class, IDatabaseObject
 		{
-			var name = $"{ControllerContext.RouteData.Values["action"]}_{ControllerContext.RouteData.Values["cutoff"]}";

```

Comment:
`ControllerContext.RouteData.Values["cutoff"]` becomes `null`. However, `cutoff` includes minutes and seconds, so caching wasn't working correctly at all.